client/reducers/documentReducer.js

```diff
@@ -0,0 +1,24 @@
+import * as types from './../actions/actionTypes';
+import initialState from './initialState';
+
+export default (state = initialState.documents, action) => {
+  switch (action.type) {
+    case types.GET_ALL_DOCUMENTS_SUCCESS:
+      return action.documents;
+
+    case types.CREATE_DOCUMENT_SUCCESS:
+      return [
+        ...state,
+        Object.assign({}, action.document)
```

Comment:
What is the use of `Object.assign({}, action.document)` here? DO you think it adds any real value here? I do not think so.